packages/arb-token-bridge-ui/src/tokenLists.tsx

```diff
@@ -74,6 +74,15 @@ export const BRIDGE_TOKEN_LISTS: BridgeTokenList[] = [
     isDefault: true,
     logoURI:
       'https://ipfs.io/ipfs/QmQAGtNJ2rSGpnP6dh6PPKNSmZL8RTZXmgFwgTdy5Nz5mx'
+  },
+  {
```

Comment:
Need to add token list for the local network for it to initiate bridge tokens

Reply:
Can we push to this array from the test itself and not keep it here all of the time?

Reply:
The list is hardcoded on the bridge side and pulled in the runtime. Don't think we can easily push it without interfering with the bridge logic.